anyway/localization.py

```diff
@@ -235,8 +237,21 @@
     "ZURAT_ISHUV": "צורת יישוב"
 }
 
-_cities = list(csv.DictReader(open(os.path.join("static/data/cities.csv"))))
-_cities_names = {int(x[field_names.sign]): x[field_names.name].decode('cp1255') for x in _cities}
+if six.PY3:
+    _open_hebrew_textfile = partial(open, encoding='cp1255')
```

Comment:
Partial is great, but why not lambda for consistency with the below?

Reply:
This is a bit different. The `_decode_hebrew` lambda calls a method on the given object, so I can't use partial.